packages/driver/src/cy/retries.ts

```diff
@@ -73,6 +73,18 @@ export const create = (Cypress, state, timeout, clearTimeout, whenStable, finish
       }).message
 
       const retryErrProps = modifyErrMsg(error, prependMsg, (msg1, msg2) => {
+        const autOrigin = Cypress.state('autOrigin')
+        const commandOrigin = window.location.origin
+
+        if (autOrigin && commandOrigin !== autOrigin) {
```

Comment:
Do we also want to check the feature flag here is well `isExperimentalMultiDomain`?

Reply:
I think this is an ok change without needing the experimental flag because without the experimental flag on, on page load will fail when a cross origin domain is accessed, we'd never get to this point.